src/ServiceProvider.php

```diff
@@ -78,4 +84,24 @@ protected function shareValidationErrors()
             });
         });
     }
+
+    protected function registerExceptionHandlers()
+    {
+        $handler = app(ExceptionHandler::class);
+
+        // The 'renderable' method is only available as of Laravel 8+
+        // In earlier version of Laravel, you'll have to register
+        // these yourself in your app's exception handler.
+        if (! method_exists($handler, 'renderable')) {
+            return;
+        }
+
+        $handler->renderable(function (HttpExceptionInterface $exception) {
+            if ($exception->getStatusCode() === 419 && $exception->getPrevious() instanceof TokenMismatchException) {
+                return Redirect::back()->with([
+                    'error' => Config::get('inertia.csrf_error', 'The page expired, please try again.'),
```

Comment:
```suggestion
                    'error' => Config::get('inertia.csrf_error', __('The page expired, please try again.')),
```

I think it would be good to also translate this text by default